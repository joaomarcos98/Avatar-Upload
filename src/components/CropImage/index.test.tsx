/** @name Dependencies*/
import { screen } from "@testing-library/react"
/** @name Components*/
import { CropImage } from "."
/** @name Utils*/
import { renderWithTheme } from "../../utils/test/renderWithTheme"

describe("Crop Image component", () => {
    it("should be renders correctly", () => {
        const { container } = renderWithTheme(
            <CropImage imageUrl="any-img" zoom="20" />
        )

        expect(screen.getByRole("img")).toHaveAttribute("src", "any-img")
        expect(container.firstChild).toMatchSnapshot()
    })
})
