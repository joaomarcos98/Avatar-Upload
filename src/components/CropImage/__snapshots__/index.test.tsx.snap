// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Crop Image component should be renders correctly 1`] = `
.c0 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 113px;
  height: 113px;
  border-radius: 72px;
  overflow: hidden;
  background-color: #FFFFFF;
  z-index: 1;
}

.c1 {
  width: 200px;
  height: 200px;
  -webkit-transform: scale(calc(20 / 20));
  -ms-transform: scale(calc(20 / 20));
  transform: scale(calc(20 / 20));
}

<div
  class="c0 crop-image"
>
  <img
    alt="Avatar image"
    class="c1"
    src="any-img"
  />
</div>
`;
